acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/dsl/signer/Signer.java

```diff
@@ -83,4 +107,17 @@ public void awaitStartupCompletion() {
     waitFor(() -> assertThat(jsonRpc.ethBlockNumber().send().hasError()).isFalse());
     LOG.info("Signer is now responsive");
   }
+
+  public void sendRawJsonRpc(
```

Comment:
would make the tests read nicer if this returned a response instead of taking a callback and had a variant that didn't require headers. i.e. something like HttpClientResponse sendRawJsonRpc(final Buffer body) would remove some of the noise from sendDisabledApiReturnsBadRequest

Reply:
updated.